### DISCUSSION TITLE
7D0EE104-5066-4FA2-BAFF-56F27E48A2E9

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#214] [General]
# Hera

Hera is a European spacecraft aimed to study the Didymos-Dimorphos system in the aftermath of NASA's DART mission in September 2022. Launched in 2024, it will reach the asteroids by December 2026, and hopefully measure the efficiency of the kinetic impact method for planetary defense

https://celestia.mobi/addon?item=7D0EE104-5066-4FA2-BAFF-56F27E48A2E9

**Comment by DaveBowman2001** [comment]
**NOTE FROM THE AUTHOR (241110):**
The Didymos-Dimorphos phase of the mission isn't yet included in this add-on as the necessary trajectory data (from [JPL HORIZONS](https://ssd.jpl.nasa.gov/horizons/app.html#/)) is not yet available to be reconstructed accurately. As of this writing, there's only enough trajectory data for _Hera_ running up until 2 December 2025. Will be updated once those were eventually published, but for now, _Hera_ will remain in heliocentric orbit indefinitely.
